### PR TITLE
feat: basic route + scaffolding of claim page

### PR DESCRIPTION
**Pull request by Tulun**
1. Create basic view for the Claim page
2. Connect to router.
3. Fix the header to properly highlight the route in the header.

### CODE REVIEW

File: src/components/Header/Header.tsx [hunk 1]
```diff
@@ -48,7 +48,16 @@ const Header: React.FC<Props> = ({ openSidebar, setOpenSidebar }) => {
       <Navigation>
         <List>
           {LINKS.map(({ href, name }) => (
-            <Item key={href} aria-selected={location.pathname === href}>
+            <Item
+              key={href}
+              aria-selected={
+                location.pathname.includes("/rewards") && href === "/rewards"
```
Comment (nicholaspai) [inline src/components/Header/Header.tsx:54]: Why is this change necessary?

**Review by nicholaspai — COMMENTED** [review]
One question to understand better

File: src/components/Header/Header.tsx (hunk 1 above)
Comment (Tulun) [reply src/components/Header/Header.tsx:54]: you want the rewards tab to have that "bubble" underneath it even for sub-routes.

Hence you gotta look (atm) if it includes "/rewards". We may need to make this more robust as we add more routes.

Comment (nicholaspai) [reply src/components/Header/Header.tsx:54]: Can't this be condensed into one || statement? I feel like double ? Operator is less readable

Comment (Tulun) [reply src/components/Header/Header.tsx:54]: I actually kind of doubt it. The actual correct way of doing this is to probably create an array of parent routes ["/rewards", "/example2"] and check to see if the href includes it.

Let me attempt that anyway.

File: src/components/Header/Header.tsx [hunk 2]
```diff
@@ -47,18 +51,25 @@ const Header: React.FC<Props> = ({ openSidebar, setOpenSidebar }) => {
...
-                  search: location.search,
-                }}
-              >
-                {name}
-              </Link>
-            </Item>
-          ))}
+          {LINKS.map(({ href, name }) => {
+            const subRoute = parentRoutes.includes(href);
+            let ariaSelected = location.pathname === href;
```
Comment (nicholaspai) [inline src/components/Header/Header.tsx:56]: nit suggestion, does this work?
```suggestion
            const ariaSelected = location.pathname === href || (subRoute && location.pathname.includes(href));
```

Comment (Tulun) [reply src/components/Header/Header.tsx]: yep that does, cool.

**Review by nicholaspai — APPROVED** [review]
LGTM
